app/carbon-fields/cf-page.php

```diff
@@ -17,87 +17,168 @@ function crb_attach_header()
-      'relation' => 'OR', // Optional, defaults to "AND"
-      array(
-        'field' => 'hero_type',
-        'value' => 'default'
-      ),
-      array(
-        'field' => 'hero_type',
-        'value' => 'full'
-      )
-    )),
-    Field::make('textarea', 'hero_title', __('Hero Title'))->set_help_text(__('The title of the hero image.'))->set_rows(2)->set_width(50)->set_conditional_logic(array(
-      'relation' => 'OR', // Optional, defaults to "AND"
-      array(
-        'field' => 'hero_type',
-        'value' => 'default'
-      ),
-      array(
-        'field' => 'hero_type',
-        'value' => 'full'
-      )
-    )),
-    Field::make('text', 'hero_kicker', __('Hero Kicker'))->set_help_text(__('Displays below the title in the hero image.'))->set_width(50)->set_conditional_logic(array(
-      'relation' => 'OR', // Optional, defaults to "AND"
-      array(
-        'field' => 'hero_type',
-        'value' => 'default'
-      ),
-      array(
-        'field' => 'hero_type',
-        'value' => 'full'
-      )
-    )),
-    Field::make('text', 'hero_link_url', __('Hero Link URL'))->set_help_text(__('Enter an URL to link the hero image.'))->set_conditional_logic(array(
-      'relation' => 'OR', // Optional, defaults to "AND"
-      array(
-        'field' => 'hero_type',
-        'value' => 'default'
-      ),
-      array(
-        'field' => 'hero_type',
-        'value' => 'full'
-      )
-    )),
-    Field::make('checkbox', 'hero_image_extended', __('Hero Image Extended'))->set_option_value('true')->set_help_text('Check to extend the hero image over the sabbath column.')->set_width(50)->set_conditional_logic(array(
-      'relation' => 'OR', // Optional, defaults to "AND"
-      array(
-        'field' => 'hero_type',
-        'value' => 'default'
-      ),
-      array(
-        'field' => 'hero_type',
-        'value' => 'full'
-      )
-    )),
-    Field::make('checkbox', 'hero_scroll_hint', __('Hero Scroll Hint'))->set_option_value('true')->set_help_text('Check to extend the hero image over the sabbath column.')->set_width(50)->set_conditional_logic(array(
-      array(
-        'field' => 'hero_type',
-        'value' => 'full'
-      )
-    )),
-    Field::make('complex', 'hero_column', __('Hero Column (3 Columns)'))->set_conditional_logic(array(
-      array(
-        'field' => 'hero_type',
-        'value' => 'column'
-      )
-    ))->add_fields(array(
-      Field::make('image', 'hero_image_column', __('Hero Image')),
-      Field::make('textarea', 'hero_title_column', __('Hero Title'))->set_help_text(__('The title of the hero image.'))->set_rows(2)->set_width(50),
-      Field::make('text', 'hero_kicker_column', __('Hero Kicker'))->set_help_text(__('Displays below the title in the hero image.'))->set_width(50),
-      Field::make('text', 'hero_link_url', __('Hero Link URL'))->set_help_text(__('Enter an URL to link the hero image.'))
-    ))->set_min(3)->set_max(3)
-  ));
+    Container
```

Comment:
This should not be part of this commit. It is already part of the `v3` branch. Perhaps a `rebase` is needed?